### PR TITLE
ref #193: remove licenseManager orphan files

### PR DESCRIPTION
**Pull request by pixeljunkie**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#193
| License       | MIT

Remove licenseManager orphan files which are not used since 6.1 and don't have any need in the core alone.

### CODE REVIEW

**Comment by umulmrum** [comment]
Do not merge yet - see https://github.com/chameleon-system/chameleon-system/issues/193#issuecomment-438638297

**Comment by pixeljunkie** [comment]
I updated the PR to delete the files now.

File: UPGRADE-6.3.md [hunk 1]
```diff
@@ -636,6 +636,7 @@ Three newly defined log channels are deprecated and only necessary for backwards
 - \TPkgCmsCoreLog
 - \TPkgCmsCoreLogCleanupCronJob
 - \TPkgCmsCoreLogMonologHandler_Database
+- \TPkgCmsLicenseManager_MTLogin
```
Comment (umulmrum) [inline UPGRADE-6.3.md]: We normally can't deprecate things retroactively. As we're a bit off-the record anyway by removing the files without deprecating beforehand, it's OK as an exception.

File: UPGRADE-6.3.md [hunk 2]
```diff
@@ -799,3 +800,4 @@ that are deprecated because they are outdated and replaced by frontend themes or
 ## Page Definitions
 
 - api.pagedef.php
+- pkgCmsLicenceManager.pagedef.php
```
Comment (umulmrum) [inline UPGRADE-6.3.md]:
```suggestion
- pkgCmsLicenseManager.pagedef.php
```

File: UPGRADE-7.0.md [hunk 3]
```diff
@@ -657,3 +657,7 @@ There were some frontend styles, images and javascript helpers located in the co
 - web_modules/MTFeedbackCore
 - web_modules/MTGlobalListCore
 - web_modules/MTNewsletterSignupCore
+
+## Page Definitions
+
+- pkgCmsLicenceManager.pagedef.php
```
Comment (umulmrum) [inline UPGRADE-7.0.md]:
```suggestion
- pkgCmsLicenseManager.pagedef.php
```

**Comment by umulmrum** [comment]
Could you solve the merge conflicts?

**Pull request closed without merging**